src/providers/ArticleProvider.jsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from "react";
+import Articles from "../pages/Articles";
+import SingleArticle from "../pages/SingleArticle";
+import axios from "axios";
+
+function ArticleProvider() {
+const [articles, setArticles] = useState([]);
+useEffect(() => {
+    axios.get(`https://nc-news-project-cf.herokuapp.com/api/articles`)
+    .then((response) => {
+        setArticles(response.data.articles)
+    })
+}, [])
```

Comment:
Could you refactor this so that isnt in an api.js file.
You can then add all your api requests in one place, as the lecture yesterday was demoing. 

means you only have to make changes in a single place